build.py

```diff
@@ -3,6 +3,7 @@
 # INFORMATION:
 # This scripts compiles the original Capstone framework to JavaScript
 
+from __future__ import print_function
```

Comment:
This line shouldn't be necessary as long as you don't alias `print` or use it as a "function".
If all you're interested is using brackets (for compatibility with Python 3), it already works out-of-the-box.

Reply:
It's not strictly necessary, but if someone decides to modify the script to have multiple arguments to print then it does matter. If we want to be maximally compatible I don't see the issue with including it.

Reply:
Oh, I just tested it and you are completely right. I thought finding side-effects of the legacy `print` behavior would be much trickier. Then it all looks good to me.